CGCharter-1727386911.html

```diff
@@ -298,10 +298,10 @@ <h2 id="chairs">
     </p>
     <p>
       Participants in this group choose their Chair(s) and can replace their
-      Chair(s) at any time using whatever means they prefer. However, if 5
-      participants, no two from the same organisation, call for an election,
-      the group must use the following process to replace any current Chair(s)
-      with a new Chair, consulting the Community Development Lead on election
+      Chair(s) at any time using whatever means they prefer. However, if any 5
+      participants, no two from the same organisation, call for an election, the
+      group must use the following process to replace any current Chair(s) with
+      a new Chair, consulting the Community Development Lead on election
```

Comment:
```suggestion
      new Chair(s), consulting the Community Development Lead on election
```

To clarify multiple Chairs would not be replaced by a single Chair